pkg/crc/preset/preset.go

```diff
@@ -37,6 +37,8 @@ func ParsePresetE(input string) (Preset, error) {
 		return OpenShift, nil
 	case OKD.String():
 		return OKD, nil
+	case Microshift.String():
+		return Microshift, nil
```

Comment:
This is preventing the earlier commit if user use preset microshift so that commit can error out. I think for this change this is much better location.

Reply:
Ah right, so that `validatePreset` errors out, I thought it could be done directly there ;)